sunpy/map/sources/mlso.py

```diff
@@ -22,18 +22,18 @@ class KCorMap(GenericMap):
     COSMO K-Cor, which was deployed to the Mauna Loa Solar Observatory (MLSO) in Hawaii in
     September 2013, replacing the aging MLSO Mk4 K-coronameter.
 
-    References
```

Comment:
This had to move to ensure that your code works?

Reply:
yes, as all the other files had notes above references, so felt it was more reasonable to do here aswell.